nucypher/network/server.py

```diff
@@ -250,6 +257,35 @@ def set_policy(self, id_as_hex, request: Request):
         # TODO: Sign the arrangement here.  #495
         return  # TODO: Return A 200, with whatever policy metadata.
 
+    def revoke_arrangement(self, id_as_hex, request: Request):
+        """
+        REST endpoint for revoking/deleting a KFrag from a node.
+        TODO: How do we want to verify that this request comes from Alice?
```

Comment:
Thoughts on this / do we have a related issue?

Reply:
Ah. That's an old TODO. :grimacing: